ledger-core/testutils/runner/logicless/service.go

```diff
@@ -259,12 +266,18 @@ func (m *ExecutionClassifyMock) Set(key string, v1 contract.MethodIsolation, v2
 }
 
 func (s ServiceMock) ExecutionClassify(execution execution.Context) (contract.MethodIsolation, error) {
-	if chunk, ok := s.classifyMapping.mapper[s.keyConstructor(execution)]; !ok {
+	key := s.keyConstructor(execution)
+	if chunk, ok := s.classifyMapping.mapper[key]; ok {
 		chunk.count++
 		return chunk.v1, chunk.v2
 	}
 
-	panic("key not found")
+	s.t.Fatalf("failed to find classify execution for key '%s'", key)
```

Comment:
what do you mean here by "classify execution"? just lang wise?

Reply:
fixed